libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/DevInfoActivity.java

```diff
@@ -53,8 +53,7 @@ protected void onCreate(Bundle savedInstanceState) {
         boolean isDarkTheme = SalesforceSDKManager.getInstance().isDarkTheme();
         setTheme(isDarkTheme ? R.style.SalesforceSDK_Dark : R.style.SalesforceSDK);
 
-        // Title / layout
-        getActionBar().setTitle(R.string.sf__dev_support_title);
```

Comment:
getActionBar() throws a NPE after the recent code changes.

Reply:
Can you use `getSupportActionBar()` instead?

Reply:
I would have to make DevInfoActivity a AppCompatActivity first. Let me give it a go.

Reply:
Done with https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2402/commits/c02cb39cbcec5ab1c9f00bedc7ae8e062ece5762